app/generic/connections/connections.py

```diff
@@ -158,11 +161,11 @@ async def create_invitation(
     return invitation
 
 
-@router.post("/accept-invitation", response_model=Connection)
+@router.post("/accept-invitation")
```

Comment:
Does the openapi definition still infer the return type correctly if we don't pass response_model?